src/app/components/icons/IconMenu.tsx

```diff
@@ -64,7 +64,7 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 const instructionLinks = {
-    icons: 'https://github.com/pxblue/icons',
+    iconFont: 'https://github.com/pxblue/icons#using-the-icon-font-in-your-application',
```

Comment:
Should we make this one an NPM link too for consistency? https://www.npmjs.com/package/@pxblue/icons